DQM/SiStripMonitorClient/python/SiStripSourceConfigTier0_cff.py

```diff
@@ -71,6 +66,11 @@
     errorReplyDcs = cms.bool( True ),
 )
 
+from Configuration.StandardSequences.Eras import eras
+if eras.stage2L1Trigger.isChosen():
+    SiStripMonitorClusterBPTX.BPTXfilter.stage2 = cms.bool(True)
+    SiStripMonitorClusterBPTX.BPTXfilter.l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis")
+    SiStripMonitorClusterBPTX.BPTXfilter.l1tExtBlkInputTag = cms.InputTag("gtStage2Digis")
```

Comment:
thanks @slava77, @makortel suggested me the same, yesterday
(when I developed this config I looked for examples used by L1 config, and I ended w/ isChosen() ....)
I've just updated the config, hope it is fine, now

I also make use of the dict and I did not specify the type, again as suggested by @makortel (thanks !)